### DISCUSSION TITLE
changing the root size, but only for the fonts

### DISCUSSION THREAD

**Discussion by rcp-novicell** [tailwindlabs/tailwindcss#11401] [Help] [answered]
Hey,

I have a client, where they have a design, where the root size of the fonts is 16px on mobile and 18px on desktop. All the others size for height, padding, etc. is set to the root size 16px. So if I change the root to 18px, the design would be larger that the received design.

So is there an easy fix for this or would I have to set all the fonts size in pixels ?

**Comment by wongjn** [comment, marked as answer]
Consider changing the font size of the `<body>` instead of `<html>`. This will still affect all text on the page but does not effect the length represented by the `rem` unit. For example, assuming "desktop" means `lg` breakpoint:

```html
<!-- `text-base` is a browser default and could be left out -->
<body class="text-base lg:text-lg">
```

**Reply by rcp-novicell** [reply]
Thanks - that solved my problem
